src/core/server_status.proto

```diff
@@ -97,21 +97,6 @@ message ModelControlRequestStats
   StatDuration success = 1;
 }
 
-//@@
-//@@.. cpp:var:: message SharedMemoryControlRequestStats
-//@@
-//@@   Statistics collected for SharedMemoryControl requests.
-//@@
-message SharedMemoryControlRequestStats
```

Comment:
I understand your earlier question  now. To avoid breaking existing code just mark this as deprecated with message indicating that it will always report 0.

Reply:
Done